src/web/hooks/use-call-tool.test.ts

```diff
@@ -41,7 +41,7 @@ describe("useCallTool - onSuccess callback", () => {
     const { result } = renderHook(() =>
       useCallTool<typeof args, typeof data>(toolName)
     );
-    act(() => {
+    await act(async () => {
```

Comment:
can you explain why this is working @harijoe 😅?
`callTool` is not async so I don't really understand this